src/databricks/labs/ucx/workspace_access/generic.py

```diff
@@ -189,6 +186,26 @@ def _safe_get_permissions(self, object_type: str, object_id: str) -> iam.ObjectP
             else:
                 raise RetryableError() from e
 
+    @retried(on=[RetryableError])
```

Comment:
No timeout configured. See how it's configurable for unit testing in other places

Reply:
Which timeout value do you suggest?
I haven't found examples on how to configure it in the unit tests.
I was rather thinking of mocking `@retried` in the unit tests, wdyt?

Reply:
@tnguyen-db did you look at this one?
https://github.com/databrickslabs/ucx/blob/bc7f29e0727c7b38d53f7c43e194fbe653bf46d6/src/databricks/labs/ucx/workspace_access/secrets.py#L92-L97